.github/workflows/test-linux.yml

```diff
@@ -2,9 +2,9 @@ name: test-linux
 
 on:
   push:
-    branches: [master]
+    branches: [master, stable-*]
```

Comment:
For my info: We decided on `stable-` vs `release-` or `maintenance-`?

Reply:
That wasn't discussed, but that was the name I chose when I made the branch. If you have a preference I can change that before merging today.

Reply:
No preference, just clarifying. Not sure what other projects use for this.